.github/workflows/continuous-integration.yml

```diff
@@ -392,6 +392,16 @@ jobs:
           GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
           MAVEN_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
           MAVEN_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+      - name: Cancel workflow
+        if: failure()
+        uses: andymckay/cancel-action@0.5
+
+  update_version:
+    runs-on: ubuntu-22.04
```

Comment:
This is not part of your PR, but I would suggest to use everywhere the same ubuntu version. The last LTS should be 24.04, but we have to check if everything works. Could you write an issue for this please?

Reply:
Yes, I could introduce a constant for the Ubuntu version and set it to 22.04, as we know that it works. I can also create an issue for the update to 24.04

Reply:
I would be in favor of a constant for ubuntu-22.04, yes